### PR TITLE
[PortfolioProfitability] Refactor

### PR DESCRIPTION
**Pull request by Herklos**

### CODE REVIEW

File: octobot_trading/portfolios/portfolio_value_holder.py [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+        :param refresh_values: when True, force origin portfolio reevaluation
+        :return: the origin portfolio current value
+        """
+        if refresh_values:
+            self.current_crypto_currencies_values.update(
+                self._evaluate_config_crypto_currencies_and_portfolio_values(self.origin_portfolio.portfolio))
+        return self._update_portfolio_current_value(self.origin_portfolio.portfolio,
+                                                    currencies_values=self.current_crypto_currencies_values)
+
+    async def _init_portfolio_values_if_necessary(self, force_recompute_origin_portfolio) -> None:
```
Comment (GuillaumeDSM) [inline octobot_trading/portfolios/portfolio_value_holder.py:114]: 👍

**Review by GuillaumeDSM — APPROVED** [review]
If tentacles strategy tests are passing, it's good for me, good job !

**Comment by Herklos** [comment]
> If tentacles strategy tests are passing, it's good for me, good job !

Yes they are :wink:

**Comment by Herklos** [comment]
> > If tentacles strategy tests are passing, it's good for me, good job !
> 
> Yes they are

Just checked and 147/147 tests are passing

**Pull request merged**